core/parcel-runtime/src/utils/0-patch-module.ts

```diff
@@ -59,6 +59,14 @@ export function getHostname() {
   return runtimeData.host
 }
 
+export function getSocketHostname() {
+  if (!runtimeData.host || runtimeData.host === "0.0.0.0") {
+    return "localhost";
```

Comment:
It's probably worth mentioning that attempting to connect to 0.0.0.0 from a content script UI results in this error:
<img width="531" alt="image" src="https://github.com/PlasmoHQ/plasmo/assets/12898578/35f687e4-e4f7-4ea8-9591-51508ecf6c94">